src/cdk-experimental/listbox/listbox.md

```diff
@@ -0,0 +1,216 @@
+  lists the invalid values
+- `cdkListboxUnexpectedMultipleValues` - Raised when a single-selection listbox is bound to a value
+  containing multiple selected options.
+
+<!-- example({
+  "example": "cdk-listbox-forms-validation",
+  "file": "cdk-listbox-forms-validation-example.ts",
+  "region": "errors"
+}) -->
+
+### Disabling options
+
+Individual options can be disabled for selection by setting `cdkOptionDisabled` on them.
+In addition, the entire listbox control can be disabled by setting `cdkListboxDisabled` on the
+listbox element.
+
+<!-- example({
+  "example": "cdk-listbox-disabled",
+  "file": "cdk-listbox-disabled-example.html",
+  "region": "listbox"
+}) -->
+
+### Accessibility
+
+The directives defined in `@angular/cdk/listbox` follow accessibility best practices as defined
+in the [ARIA spec][aria]. Keyboard interaction is supported as defined in the
+[ARIA listbox keyboard interaction spec][keyboard] _without_ the optional selection follows focus
+logic (TODO: should we make this an option?).
+
+#### Listbox label
+
+Always give the listbox a meaningful label for screen readers. If your listbox has a visual label,
+you can associate it with the listbox using `aria-labelledby`, otherwise you should provide a
+screen-reader-only label with `aria-label`.
+
+#### Roving tabindex vs active descendant
+
+By default, the CDK listbox uses the [roving tabindex][roving-tabindex] strategy to manage focus.
+If you prefer to use the [aria-activedescendant][activedescendant] strategy instead, set
+`useActiveDescendant=true` on the listbox.
+
+<!-- example({
+  "example": "cdk-listbox-activedescendant",
+  "file": "cdk-listbox-activedescendant-example.html",
+  "region": "listbox"
+}) -->
+
+#### Orientation
+
+Listboxes assume a vertical orientation by default, but can be customized by setting the
+`cdkListboxOrientation` input oe listbox. Note that this only affects the keyboard navigation. You
+will still need to adjust your CSS styles to change the visual appearance.
+
+<!-- example({
+  "example": "cdk-listbox-horizontal",
+  "file": "cdk-listbox-horizontal-example.html",
+  "region": "listbox"
+}) -->
+
+#### Option typeahead
+
+The CDK listbox supports typeahead based on the option text. If the typeahead text for your options
+needs to be different than the display text (e.g. to exclude emoji), this can be accomplished by
+setting the `cdkOptionTypeaheadLabel` on the option.
+
+<!-- example({
+  "example": "cdk-listbox-custom-typeahead",
+  "file": "cdk-listbox-custom-typeahead-example.html",
+  "region": "listbox"
+}) -->
+
+#### Keyboard navigation options
+
+When using keyboard navigation to navigate through the options, the navigation wraps when attempting
+to navigate past the start or end of the options. To change this, set
+`cdkListboxNavigationWraps=false` on the listbox.
+
+Keyboard navigation skips disabled options by default. To change this set
+`cdkListboxNavigationSkipsDisabled=false` on the listbox.
```

Comment:
Hmm, what about something like`cdkListboxNavigateDisableOptions` so it can be a flag attribute? Very hard to think of a solid name here